### PR TITLE
Redirect website to HTTPS

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

**Comment by garbas** [comment]
I enabled the HTTPS via Netlify, but we are still missing the redirect. I'm hope this does the trick.

File: website/netlify.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[[redirects]]
+  from = "https://nickel-lang.netlify.com/*"
+  to = "https://nickel-lang.org/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "http://nickel-lang.netlify.com/*"
+  to = "http://nickel-lang.org/:splat"
```
Comment (yannham) [inline website/netlify.toml:9]: Is it on purpose that the target is `http` and not `https`?

**Review by yannham — APPROVED** [review]
Thanks! LGTM, although please look at the comment to make sure there isn't a missing `https`.

File: website/netlify.toml (hunk 1 above)
Comment (garbas) [reply website/netlify.toml]: Fixed. Good catch!

**Pull request merged**